### ISSUE TITLE
Build issue on Debian 8

### ISSUE THREAD

**Issue by metatoaster** [cmlibs/zinc#44]
When trying to build according to instructions at http://physiomeproject.org/software/opencmiss/zinc/documentation/support/build on Debian 8, 64-bit. (had to point to `cmake-2.8` to `cmake-3.0` as Debian 8 ships cmake 3.0), the compilation ends with this.

```
Linking CXX shared library libzinc.r11339.so
/usr/bin/ld: /tmp/tmpE4eQhsbuildout-zinc/dependencies/install/lib/InsightToolkit/libITKEXPAT.a(xmlparse.o): relocation R_X86_64_32 against `XmlInitUnknownEncoding' can not be used when making a shared object; recompile with -fPIC
/tmp/tmpE4eQhsbuildout-zinc/dependencies/install/lib/InsightToolkit/libITKEXPAT.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
core/CMakeFiles/zinc.dir/build.make:4902: recipe for target 'core/libzinc.r11339.so.3.0.1' failed
make[2]: *** [core/libzinc.r11339.so.3.0.1] Error 1
CMakeFiles/Makefile2:110: recipe for target 'core/CMakeFiles/zinc.dir/all' failed
make[1]: *** [core/CMakeFiles/zinc.dir/all] Error 2
Makefile:137: recipe for target 'all' failed
make: *** [all] Error 2
```

I had applied the suggestion on the usage of `-fPIC` (by calling cmake with the arguments `-DCMAKE_C_FLAGS=-fPIC -DCMAKE_CXX_FLAGS=-fPIC`) and verified that the flags are being used, but compilation still fails at the end.  That said inspecting the `libITKEXPAT.a` archive I noted that the `xmlparse.o` object file doesn't actually seem to indicate that it's compiled with `fPIC` (as per [this SO post](http://stackoverflow.com/questions/19768267/relocation-r-x86-64-32s-against-linking-error)).

Also I must note, this is on the old SVN repo, so those instructions may need updating for this git repository if this is the current development.

**Comment by hsorby** [comment]
You are correct here these instructions are for the old svn repository which while still in existence is no longer maintained.  The new build instructions that should be followed are here:

http://opencmiss.org/documentation/cmake/docs/#building-opencmiss-from-source
